src/tools/rbd_mirror/RemotePoolPoller.cc

```diff
@@ -80,21 +80,20 @@ void RemotePoolPoller<I>::get_mirror_uuid() {
 template <typename I>
 void RemotePoolPoller<I>::handle_get_mirror_uuid(int r) {
   dout(10) << "r=" << r << dendl;
-  std::string remote_mirror_uuid;
+  std::string mirror_uuid;
```

Comment:
Not sure I'm following: `RemotePoolPoller::get_mirror_uuid()` operates on `m_remote_io_ctx` (i.e. a remote RADOS client) so it would seem that it can't get anything but a remote mirror UUID here?

Reply:
Right! That should be correct. I might have simply got carried away with the namings  `mirror_uuid`, `mirror_peer_uuids`, `remote_mirror_peer_uuid` ...

If I find any other evidences I will get here.  For now I will close it.

Sorry for the noise.